### PR TITLE
Remove Summarization Tasks and Related Code

### LINKED ISSUE #2651: Remove summarization tasks and apis
https://github.com/ohcnetwork/care/blob/dc578c7cbce5b66cf5121cbbc9fbd5e44c1d0133/care/facility/tasks/__init__.py#L26-L55

Remove the following stuff for each of the above tasks:
- task definition
- api endpoints
- serializers and views
- settings and mentions in docs

### PR DESCRIPTION
**Pull request by areebahmeddd**
## Proposed Changes

- Removed periodic summarization tasks from `tasks/__init__.py`.
- Removed all summarization task definitions from `tasks/summarisation.py`.
- Removed summarization task settings from `config/settings/base.py`.
- Removed mentions of summarization tasks from documentation.
- Removed summarization task serializers from `api/serializers/summary.py`.
- Removed summarization task viewsets from `api/viewsets/summary.py`.
- Removed summarization task API routes from `config/api_router.py`.

### Associated Issue

- Closes #2651. The proposed changes remove all summarization tasks, their definitions, related API endpoints, serializers, views, models, settings, and mentions in the documentation as requested in the issue.

## Merge Checklist

- [ ] Tests added/fixed
- [ ] Update docs in `/docs`
- [ ] Linting Complete
- [ ] Any other necessary step

@ohcnetwork/care-backend-maintainers @ohcnetwork/care-backend-admins

## Summary by CodeRabbit

- **Bug Fixes**
	- Removed several classes and methods related to facility and district data summarization.
	- Eliminated periodic task registrations for summarization tasks.

- **Documentation**
	- Updated documentation to reflect the removal of environment variable configurations related to summarization tasks.

- **Chores**
	- Streamlined codebase by removing unused serializers, viewsets, and Celery tasks.
	- Removed obsolete models and their associated logic from the application.

### CODE REVIEW

File: care/facility/api/viewsets/summary.py
Comment (sainak) [inline care/facility/api/viewsets/summary.py:1]: this file can be removed, just the filters are remaining which are not used by any other views

**Review by sainak — APPROVED** [review]

**Comment by areebahmeddd** [comment]
@sainak Seems like you've done the remaining changes? thanks haha :)
I did see several unused blocks of code, on searching those objects throughout codebase, i found several files were interlinked. 

Decided best to ask in discussion after making pr

**Comment by sainak** [comment]
> @sainak Seems like you've done the remaining changes? thanks haha :)

yeah, there was some other stuff that needed to be removed, and it was easier this was

**Comment by sainak** [comment]
@coderabbitai review

**Review by vigneshhari — APPROVED** [review]

**Pull request merged**